go-controller/hack/update-codegen.sh

```diff
@@ -9,20 +9,6 @@ if [ -z "${crds}" ]; then
   exit
 fi
 
-if  ! ( command -v controller-gen > /dev/null ); then
```

Comment:
Any reason in particular that you removed this?

Reply:
@alexanderConstantinescu removed it because in a previous PR https://github.com/ovn-org/ovn-kubernetes/pull/1475 client-go was updated and the restriction on the controller-gen version is removed

Reply:
put the download of controller-gen back in the script